src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -9,30 +9,33 @@
 log = Logger(__name__)
 
 
-def _get_project_messages_from_engagement_db(analysis_configurations, engagement_db, cache_path):
+def _get_project_messages_from_engagement_db(analysis_configurations, engagement_db, cache_path=None):
     """
-
-    Downloads project messages from engagement database. It performs a full download if there is no previous export and
-    incrementally otherwise.
+    Gets project messages from engagement database.
```

Comment:
I actually prefer the old description (with previous export replaced with cache path)

Reply:
I've updated the description, thanks.